src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.android.cs

```diff
@@ -81,12 +85,25 @@ public partial class MediaManager : Java.Lang.Object, IPlayer.IListener
 		}
 	}
 
+	static Bitmap? CreateBitmap(int width, int height, Bitmap.Config config)
+	{
+		if (OperatingSystem.IsAndroidVersionAtLeast(26))
+		{
+			return Bitmap.CreateBitmap(width, height, config, true);
+		}
+		else if (Build.VERSION.SdkInt <= BuildVersionCodes.O)
```

Comment:
Changing this will result in a message on build about versioning. I would prefer to not have the errors. All of the changes and choices with regards to what API I use is to get rid of actual errors that will be present using the other API. This API is newer and provided by Microsoft. It does not have the required flexibility for all situations. I would prefer to not change this. Changing either of these to the other one will cause a warning message about compatibility. I can of course just use the one that will work and add suppression's where it the compiler is wrong?

Reply:
Now I will admit to my error here. I was wrong! You can do what you suggested. My bad. I did test it but for what ever reason it never occured to me to use `!OperatingSystem.IsAndroidAtLeast(26)`